lib/game.rb

```diff
@@ -0,0 +1,49 @@
+class Game
+    include Interface
+
+    def initialize(player1, player2, board)
+        @player1 = player1
+        @player2 = player2
+        @board = board
+        @status = "initial"
+    end
+
+    def play
+        getInput('welcome')      
```

Comment:
Is there a way we can transfer the call to `getInput` somewhere else and not here in the `play` method? :)

Reply:
yes, same put it to `main.rb`.